packages/materialdesign-components/src/Button/Button.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import { setContext, getContext } from "svelte";
+  import Icon from "../Icon.svelte";
+  import ripple from "../Ripple.js";
+  import ClassBuilder from "../ClassBuilder.js";
+
+  const cb = new ClassBuilder("button", ["primary", "medium"]);
+
+  export let variant = "raised";
+  export let colour = "primary";
+  export let size = "medium";
+
+  export let href = "";
+  export let icon = "";
+  export let trailingIcon = false;
+  export let fullBleed = false;
+
+  export let text = "";
+  export let disabled = false;
+
+  $: if (!!icon) {
```

Comment:
You don't need to use `!!` here. JavaScript will automatically coerce your value to a boolean whenever it's wrapped in an if, or used in a conditional expression like below.

Reply:
Just pushed to update this.